docs/src/examples/components/Menu/Variations/MenuExampleVerticalFluid.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import _ from 'lodash'
+import { Menu, MenuItem } from '@stardust-ui/react'
+
+const items = [
+  { key: 'editorials', content: 'Editorials' },
+  { key: 'review', content: 'Reviews' },
+  { key: 'events', content: 'Upcoming Events' },
+]
+
+class MenuExampleVerticalFluid extends React.Component {
+  state = { activeIndex: 0 }
+
+  handleItemClick = activeIndex => () => {
+    this.setState({ activeIndex })
+  }
+
+  render() {
+    const { activeIndex } = this.state
+
+    return (
+      <Menu vertical fluid>
+        {_.times(3, i => {
```

Comment:
according to our agreements, we should use explicit way of declaring components in the examples (although totally understand your reasons:)

Reply:
You mean write html instead of using javascript/lodash?

Reply:
yes, exactly